dorado/scheduling/orbit.py

```diff
@@ -110,3 +113,19 @@ def __call__(self, time):
         else:
             coord = coord[0]
         return coord
+
+    def is_night(self, time):
+        """Determine if the spacecraft is in orbit night.
```

Comment:
I love the term "orbit night".

Reply:
Hey, it's what our sciops people call it.